examples/operators/README.rst

```diff
@@ -0,0 +1,7 @@
+
+.. _operators_examples:
+
+Operators Examples
```

Comment:
I realize I previously went against the conventional title case that you use here, but it's actually more legitimate as we write everything in English. We should make titles aligned with this title case a bit everywhere